### DISCUSSION TITLE
No Display Waveshare 1.44" LCD Hat

### DISCUSSION THREAD

**Discussion by nevwain** [WLAN-Pi/feedback#24] [Hardware - WLAN Pi R4] [answered]
Hello

I have just flashed the latest WLANPi CE OS onto my Pi4 8Gb.  This is my first attempt at the CE edition.  The LCD panel backlights but doesn't display any FPMS.  Do I have to SSH in and alter anything to get this to work or should it work straight from the existing image?

Additionally - how long would you expect a fresh install to take to completely boot and produce a FPMS?  I left it about 5 mins before pulling the power and reflashing the image.

I've got a PiJuice Hat on order so may wait for that to arrive before trying again.

Thanks for a fantastic project!

Regards

--

Nev

**Comment by wifinigel** [comment, marked as answer]
Hi Nev,

Try the following and then reboot the RPi:

```sudo touch /boot/waveshare```

Regards

Nigel.
